ksqldb-engine/src/main/java/io/confluent/ksql/planner/plan/QueryFilterNode.java

```diff
@@ -358,6 +363,27 @@ public Void visitComparisonExpression(
       }
     }
 
+    @Override
+    public Void visitLikePredicate(final LikePredicate node, final Object context) {
+      if (node.getValue() instanceof UnqualifiedColumnReferenceExp) {
+        final UnqualifiedColumnReferenceExp column = (UnqualifiedColumnReferenceExp) node.getValue();
+        final ColumnName columnName = column.getColumnName();
+        final Column col = schema.findColumn(columnName)
```

Comment:
Since we're effectively doing the type checking here since it's not done in `ExpressionTypeManager`, we should also verify that the column is a string type too, unless we actually automatically coerce the type, but I don't think we do.  Might want to test that.

Reply:
Added string type column verification.